scripts/ec2-types.txt

```diff
@@ -43,6 +43,12 @@ m4.2xlarge
 m4.4xlarge
 m4.large
 m4.xlarge
+m5.12xlarge
```

Comment:
This list is used to do some testing.
Some regions do not have m5/c5 type yet.

Reply:
I'm just looking to be able to use m5 instances on the Rancher server UI, but to be honest I'm not sure how to achieve that best.

Reply:
oh, this project is used to build RancherOS.
you can file an issue to `https://github.com/rancher/rancher/issues`.